pkg/routeagent_driver/handlers/ovn/handler.go

```diff
@@ -139,6 +141,14 @@ func (ovn *Handler) RemoteEndpointCreated(endpoint *submV1.Endpoint) error {
 	ovn.mutex.Lock()
 	defer ovn.mutex.Unlock()
 
+	lastProcessedTime, ok := ovn.remoteEndpointTimeStamp[endpoint.Spec.ClusterID]
+
+	if ok && lastProcessedTime.After(endpoint.CreationTimestamp.Time) {
+		logger.Infof("Ignoring new remote %#v since a later endpoint was already"+
+			"processed", endpoint)
+		return nil
+	}
```

Comment:
We already have a `remoteEndpoints` map which stores the entire struct - can't we use that instead of introducing a new map that just stores the time stamp?